packages/backend/src/managers/playground.ts

```diff
@@ -205,6 +207,26 @@ export class PlayGroundManager {
       }
     });
 
+    let contacted = false;
+    const start = Date.now();
+    while (
+      Date.now() - start < STARTING_TIME_MAX &&
+      !contacted &&
+      this.playgrounds.get(modelId).status === 'starting'
+    ) {
+      try {
+        const response = await fetch(`http://localhost:${freePort}`);
+        contacted = response.ok;
```

Comment:
Some sleep between two tries would be less resource consuming

Reply:
Added